File: MACHINE RUN 06-11-2020/Avearge of 5 nos/average of 5 numbers.c
#include<iostream>
using namespace std;
class Average
{
    int n1,n2,n3,n4,n5,average;
public:
    void Display()
    {
        cout<<"Enter the First Number";
        cin>>n1;
        cout<<"Enter the Second Number";
        cin>>n2;
        cout<<"Enter the Third Number";
        cin>>n3;
        cout<<"Enter the Fourth Number";
        cin>>n4;
        cout<<"Enter the Fifth Number";
        cin>>n5;
        cout<<"The avg is"<<(n1+n2+n3+n4+n5)/5;

    }};
    int main()
    {
        Average a1;
        a1.Display();
    }

File: ASSIGNMENT 2(Java)/averageof5numbers.java
import java.util.Scanner;
class Average
{
    public static void main(String args[])
    {
        int num1,num2,num3,num4,num5,avg;
        System.out.println("Enter First Number : ");
        Scanner sc = new Scanner(System.in);
        num1 = sc.nextInt();
        System.out.println("Enter Second Number : ");
        num2 = sc.nextInt();
        System.out.println("Enter Third Number : ");
        num3 = sc.nextInt();	
        System.out.println("Enter Fourth Number : ");
        num4 = sc.nextInt();
        System.out.println("Enter fifth Number : ");
        num5 = sc.nextInt();
       avg=(num1+num2+num3+num4+num5)/5;
       if(avg>10)
   {
System.out.println("is greater than 10");
   } 	
else
{
System.out.println("is not greater than 10");	
}

    }
}
 